### ISSUE TITLE
 Bug, osticket 1.10: setting „Recipients“ for „Alerts and Notices“ to „No one (disable Alerts and Notices)“ in a department will not prevent sending alerts and notices

### ISSUE THREAD

**Issue by mfelber** [osTicket/osTicket#3937]
### Prerequisites

* [y ] Can you reproduce the problem in a fresh installation of the "develop" branch?
* [n] Do you have any errors in the PHP error log, or javascript console?
* [y] Did you check the [osTicket forums](http://osticket.com/forum/categories)?
* [y] Did you [perform a cursory search](https://github.com/osTicket/osTicket/issues) to see if your bug or enhancement is already reported?

### Description

If „Recipients“ for „Alerts and Notices“ is set to „No one (disable Alerts and Notices)“ in a department it will not prevent sending alerts and notices. 
Alerts like for example the "New Internal Activity Alert" or "Ticket Assignment Alert" or "New Ticket Notice" will still be sent to ticket of this department.
### Steps to Reproduce

1. Set the recipients for "Alerts and Notices" to „No one (disable Alerts and Notices)“ and save the changes:

![disabled setting](https://user-images.githubusercontent.com/5539456/29743205-c258aed6-8a8c-11e7-99b4-14171c7b994e.png)

2. Create a ticket for this department and assign it to an agent
3. The agent and user are still getting notified with alerts and notices.

**Expected behavior:** This setting should disable alerts and notices for this department.

**Actual behavior:** It doesn't disable alerts and notices for this department.

### Versions

Server Information
osTicket Version | 1.8-git (?)  Upgrade— 901e5ea is available Web Server Software | Apache MySQL Version | 5.6.26 PHP Version | 5.6.12
gdlib | Used for image manipulation and PDF printing imap | Used for email fetching xml | XML API xml-dom | Used for HTML email processing json | Improves performance creating and processing JSON mbstring | Highly recommended for non western european language content phar | Highly recommended for plugins and language packs intl | Highly recommended for non western european language content fileinfo | Used to detect file types for uploads APCu | Improves overall performance Zend Opcache | Improves overall performance
cgi.fix_pathinfo | "1" is recommended if AJAX is not working date.timezone | Europe/Berlin
Schema | osticket110 (localhost) Schema Signature | 98ad7d550c26ac44340350912296e673 Space Used | 2.45 MiB Space for Attachments | 0.01 MiB Timezone | CEST (Interpreted as Europe/Berlin)
OS: ubuntu 14.04 Browser: Chrome 60.0.3112.101, Safari 10.1.2 (12603.3.8)

**Comment by JediKev** [comment]
Are you trying to make the "Default" Department Private? If so that's why it's throwing the error. You cannot make a Default Department Private 😃

**Comment by mfelber** [comment]
I think you mean my other issue report. In this one there is no error. It's about different thing 😉

**Comment by JediKev** [comment]
Oops! Sorry! Yes, I meant the other one...I have too many tabs open 🙄

**Comment by JediKev** [comment]
This seems to be working just fine for me...
Check **Admin Panel > Emails > Settings > Admin's Email Address** and see if it's the same email address as the one that's still getting the alerts. Let me know!

**Comment by mfelber** [comment]
Yes, I have checked that the admin address is not in use for the affected agents.
I also added and changed some things in my issue report so you can better understand how to replicate the issue.
